sample_code/ep102/rev04/t.py

```diff
@@ -0,0 +1,39 @@
+class User(tuple):
+    _fields = ('name', 'age', 'location')
+
+    def __new__(cls, name, age, location):
+        return tuple.__new__(cls, (name, age, location))
+
+    @property
+    def name(self):
+        return self[0]
+
+    @property
+    def age(self):
+        return self[1]
+
+    @property
+    def location(self):
+        return self[2]
+
+    def __repr__(self):
+        return (
+            f'{type(self).__name__}('
+            f'name={self.name!r}, age={self.age!r}, location={self.location!r}'
```

Comment:
Added [missing ](https://youtu.be/sfDSQSj-uvQ?t=744) `!r` formatting -- I believe that this was the intention.